examples/apfl_example/client.py

```diff
@@ -40,11 +42,29 @@ def get_criterion(self, config: Config) -> _Loss:
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="FL Client Main")
     parser.add_argument("--dataset_path", action="store", type=str, help="Path to the local dataset")
-
+    parser.add_argument(
+        "--server_address",
+        action="store",
+        type=str,
+        help="Server Address for the clients to communicate with the server through",
+        default="0.0.0.0:8080",
+    )
+    parser.add_argument(
+        "--seed",
+        action="store",
+        type=int,
+        help="Seed for the random number generator",
+        required=True,
```

Comment:
Can we make this an optional parameter rather than required throughout? We have it as an optional argument for the clients so I think that's reasonable.

Reply:
I'm also good with using a default parameter if we would rather.

Reply:
I'm also good with using a default parameter if we would rather.

Reply:
Actually, I am not sure which way to go is better. So if they want to use warm up and first run them, it is good to have it as required so they wouldn't forget to fix it. On the other hand I agree that it is not necessary if they run a vanilla version so is should not be required. Maybe I can have assertion in beginning as if they provide warm_up_dir they should set a seed?

Reply:
With the new setting of experiments I think this can be the best approach, previously I wanted to keep the setting of experiment in the paper.